setup.py

```diff
@@ -39,7 +39,7 @@ def read(*parts):
     description='Linux Distribution - a Linux OS platform information API',
     long_description=read('README.rst'),
     py_modules=['distro'],
-    install_requires=['six'],
+    install_requires=[],
```

Comment:
I'm glad you removed the dependency. you can also just remove the `install_requires` argument then.